app/router.js

```diff
@@ -203,17 +203,6 @@ Router.map(function () {
 
   this.route('profile');
 
-  this.route('quickpost');
-
-  this.route('room-adverts', function () {
-    this.route('new');
-
-    this.route('room-advert', { path: ':id' }, function () {
-      this.route('edit');
-      this.route('destroy');
-    });
-  });
```

Comment:
Not a _huge_ fan of removing the routes for the room-adverts we just added. I suspect this may have gone wrong during a merge? or you accidentally removed this. 
Either way, I've added it back.

Reply:
This probably went wrong during a merge I did not mean to do this